native/07_node-operation/10_basics/index.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Getting Started
```

Comment:
Is it meant to be getting started, or basics?

Reply:
When learning a new technology you typically start with the basics, so I think they can be used interchangeably. Basics is more concise. I can change it back to getting started though, just let me know.